mozartdata/transforms/dev_reporting/goodr_com_usage_kpis.sql

```diff
@@ -0,0 +1,145 @@
+/*
+    This table shows the daily KPIs for goodr.com usage.
+    Columns:
+        date:
+            Date reported on. This comes from dim.date to prevent issues
+            with a date not existing in Shopify.
+        sessions:
+            Total sessions as defined by Shopify for each day.
+        users:
+            Labeled "Online store visitors" in Shopify, this is the number of
+            unique users that visited the site wach day regardless of their
+            session count.
+        total_customers:
+            This is the total number of users that bought a product in a day.
+    Todos:
+        - handle leap years
+ */
+
+with
+    marketing_totals as (
+                            select
+                                date::date           as date
+                              , round(sum(spend), 2) as total_spend
+                              , sum(impressions)     as total_impressions
+                            from
+                                goodr_reporting.performance_media as pm
+                            where
+                                  lower(pm.account_country) = 'usa'
```

Comment:
Why are you lower casing this? The values are USA, CAN and Other. So can't we just leave it and look for USA?
```suggestion
                                  pm.account_country ='USA'
```